.github/workflows/scripts/slack_gem_notifications/README.md

```diff
@@ -1,14 +1,19 @@
-# Slack Gem Notifications
+# Slack Notifications
 
-An internal tool for alerting the Ruby agent team of Ruby gem version updates. 
+An internal tool for alerting the Ruby agent team of gem version updates and Ruby CVEs. 
 
 ## Description
 
-This script runs every 24 hours via GitHub Actions and uses the RubyGems API to see if gems were updated during that time. A slack alert is sent to the Ruby agent team's notification channel when an update is found. This alert contains the updated gem name, the gem's previous and latest version, and how to view more information on the update.
+This script runs every 24 hours via GitHub Actions and sends notifications for 1) gem version updates and 2) Ruby security vulnerabilities.
 
+Gem Version Updates
+The RubyGems API is used to see if gems were updated during that time. A slack alert is sent to the Ruby agent team's notification channel when an update is found. This alert contains the updated gem name, the gem's previous and latest version, and how to view more information on the update.
```

Comment:
"Slack" ought to be capitalized everywhere.

Reply:
Made both these changes as well as the other README changes you suggested, Kayla!